docs/semconv/database.md

```diff
@@ -78,24 +81,36 @@ All metrics in `db.instance` instruments should be attached to a [Instance resou
+| `db.cpu.utilization`        | The percentage of used CPU.                 | 1    | Gauge            | Double     | Recommended       |                                         |                   |
+| `db.mem.utilization`        | The percentage of used memory               | 1    | Gauge            | Double     | Recommended       |                                         |                   |
+| `db.tablespace.size`        | The size (in bytes) of the tablespace.      | By   | UpDownCounter    | Int64      | Optional          | `tablespace_name`(Required) identifier. | {tablespace_name} |
+| `db.tablespace.used`        | The used size (in bytes) of the tablespace. | By   | UpDownCounter    | Int64      | Optional          | `tablespace_name`(Required) identifier. | {tablespace_name} |
+| `db.tablespace.utilization` | The used percentage of the tablespace.      | 1    | Gauge            | Double     | Optional          | `tablespace_name`(Required) identifier. | {tablespace_name} |
+| `db.tablespace.max`         | The max size (in bytes) of the tablespace.  | By   | UpDownCounter    | Int64      | Optional          | `tablespace_name`(Required) identifier. | {tablespace_name} |
+| `db.disk.writes`            | Actual number of writes to disk             | By   | UpDownCounter    | Int64      | Optional          |                                         |                   |
+| `db.disk.reads`             | Actual number of reads to disk              | By   | UpDownCounter    | Int64      | Optional          |                                         |                   |
 
 ## Maintenance
 
 | Name                   | Description                                                    | Unit | Instrument Type | Value Type | Requirement Level | Attribute Key(s)    | Attribute Values |
 |------------------------|----------------------------------------------------------------|------|-----------------|------------|-------------------|---------------------|------------------|
 | `db.backup.cycle`      | Backup cycle.                                                  | s    | Gauge           | Int64      | Recommended       |                     |                  |
 
+
+## Settings
+| Name                          | Description                                                                                   | Unit | Instrument Type | Value Type | Requirement Level | Attribute Key(s)                       | Attribute Values    |
+|-------------------------------|-----------------------------------------------------------------------------------------------|------|-----------------|------------|-------------------|----------------------------------------|---------------------|
+| db.database.log.enabled       | Database logging is enabled or not. 1 (Active), 0 (Inactive).                                 | By   | Gauge           | Int        | Optional          | db.database.name(Required) identifier. | {db.database.name}  |
```

Comment:
Please change the Unit of `db.database.log.enabled`from `By` to `1`.
Also, I thought the `db.database.name`, `db.database.owner`, `db.database.create_on` should be the members of DataPointAttributes here. Same to `db.database.buff.log.enable`, `db.database.ansi.compliant`, `db.database.nls.enabled `, `db.database.case.insensitive`.

Reply:
>Please change the Unit of db.database.log.enabledfrom By to 1.

Sure.


>Also, I thought the db.database.name, db.database.owner, db.database.create_on should be the members of DataPointAttributes here

DB Name is unique and you can't have same db name twice and even if you modify the name it will be unique only. 
Still if you feel like it is needed let me know.